### PR TITLE
Cypress/E2E: fix failed tests

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Screen Shot 2021-08-24 at 10 47 20 AM
Screen Shot 2021-08-24 at 12 52 01 PM
Screen Shot 2021-08-24 at 3 54 01 PM
Screen Shot 2021-08-24 at 3 16 03 PM

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/channel/user_to_channel_admin_member_updates_manage_channel_members_modal_spec.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ describe('Change Roles', () => {
         });
     });
 
-    it('MM-10858 - Going from a Channel Member to Channel Admin update view member modal without refresh', () => {
+    it('MM-T4174 User role to channel admin/member updates channel member modal immediately without refresh', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/channel/user_to_channel_admin_member_updates_manage_channel_members_modal_spec.js:74]: Updated test ID only. This fails on daily due to https://mattermost.atlassian.net/browse/MM-38104

File: e2e/cypress/integration/enterprise/system_console/openid/openid_spec.js [hunk 2]
```diff
@@ -136,94 +128,8 @@ describe('System Console OpenId Connect', () => {
         });
         verifyOAuthLogin('Office 365', '', Cypress.config('baseUrl') + '/oauth/office365/login');
     });
-
-    it('MM-T3501 - Test Migrate from OAuth', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/enterprise/system_console/openid/openid_spec.js:140]: This test is for OAuth 2.0 which is recently deprecated.

File: e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js [hunk 3]
```diff
@@ -1,290 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```
Comment (saturninoabril) [inline e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js:1]: Tests here were split into 2 specs for ease of maintenance and make the test faster by preventing unnecessary URL visit to a channel that cause prolong page reload every time.

**Review by furqanmlk — APPROVED** [review]
Thanks @saturninoabril 
image
image
image
image

**Pull request merged**